src/o365/graph/commands/planner/planner-task-list.ts

```diff
@@ -0,0 +1,102 @@
+import auth from '../../GraphAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import { GraphItemsListCommand } from '../GraphItemsListCommand';
+import { Task } from './Task';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+
+}
+
+class GraphPlannerTaskListCommand extends GraphItemsListCommand<Task> {
+  public get name(): string {
+    return `${commands.PLANNER_TASK_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Lists Planner tasks of the current logged in user.';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    return telemetryProps;
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    let endpoint: string = "/me/planner/tasks";
+
+    endpoint = `${auth.service.resource}/v1.0${endpoint}`;
+
+    this
+      .getAllItems(endpoint, cmd, true)
+      .then((): void => {
+        if (args.options.output === 'json') {
+          cmd.log(this.items);
+        }
+        else {
+          cmd.log(this.items.map(t => {
+            const task: any = {
+              id: t.id,
+              title: t.title,
+              startDateTime: t.startDateTime,
+              dueDateTime: t.dueDateTime,
+              completedDateTime: t.completedDateTime
+            };
+            return task;
+          }));
+        }
+
+        if (this.verbose) {
+          cmd.log(vorpal.chalk.green('DONE'));
+        }
+        cb();
+      }, (err: any): void => this.handleRejectedODataJsonPromise(err, cmd, cb));
+  }
+
+  public options(): CommandOption[] {
```

Comment:
Since we're not defining any options specific for this command, this is not necessary.